apps/builder/app/dashboard/projects/project-card.tsx

```diff
@@ -118,7 +118,13 @@ const Menu = ({
           <MenuIcon width={15} height={15} />
         </IconButton>
       </DropdownMenuTrigger>
-      <DropdownMenuContent align="end">
+      <DropdownMenuContent
+        align="end"
+        css={{
+          // without z-index, menu is obscured by other projects' cards
+          zIndex: theme.zIndices[1],
```

Comment:
I think we should use DropdownMenuPortal most of the time to avoid using zIndex

Reply:
Switched to portal